docs/notebooks/inequality_constraints.pct.py

```diff
@@ -146,6 +148,105 @@ def create_bo_model(data):
 )
 plt.show()
 
+# %% [markdown]
+# ## Define a batch acquisition rule
+#
+# It is sometimes beneficial to query several points at a time instead of one. We show here how to create ad-hoc extensions of the previous acquistion function and rule.
+
+# %%
+from trieste.acquisition.function import BatchAcquisitionFunctionBuilder, BatchReparametrizationSampler
+
+
+class BatchExpectedConstrainedImprovement(BatchAcquisitionFunctionBuilder):
+    def __init__(
+        self,
+        objective_tag,
+        constraint_builder,
+        sample_size,
+        threshold,
+    ):
+        self._objective_tag = objective_tag
+        self._constraint_builder = constraint_builder
+        self._min_feasibility_probability = 0.5
+        self._sample_size = sample_size
+        self._jitter = 1e-6
+        self.threshold = threshold
+
+    def prepare_acquisition_function(self, datasets, models):
+        objective_model = models[self._objective_tag]
+        objective_dataset = datasets[self._objective_tag]
```

Comment:
there's a mix of `self._objective_tag`, `"OBJECTIVE"` and `OBJECTIVE`. Can we just use `OBJECTIVE`?